### PR TITLE
Document local Dgraph instance setup

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -54,3 +54,17 @@ based on changes in model.
...
+
+You can use `docker-compose` present in this repository to run a local Dgraph instance. It does not use TLS certificates (so you must not to provide `GRAPH_TLS_PATH` environment variable).
+
+.. code-block:: console
+
+  $ docker-compose up
+
+After running the command above (make sure your big fat daemon is up using `systemctl start docker`), you should be able to access a local Dgraph instance at `localhost:9080`. This is also the default configuration for Dgraph's adapter - you don't need to provide `GRAPH_SERVICE_HOST` explicitly.
+
+The provided `docker-compose` has also Ratel enabled for to have an UI for graph database content. To access it visit `http://localhost:8000/ <https://localhost:8080>`_.
```
Comment (CermakM) [inline README.rst]: I think there is a typo in the second link, should point to `https://localhost:8000`.

File: README.rst [hunk 2]
```diff
@@ -54,3 +54,17 @@ based on changes in model.
...
+Running Dgraph locally
+======================
+
+You can use `docker-compose` present in this repository to run a local Dgraph instance. It does not use TLS certificates (so you must not to provide `GRAPH_TLS_PATH` environment variable).
+
+.. code-block:: console
+
+  $ docker-compose up
+
+After running the command above (make sure your big fat daemon is up using `systemctl start docker`), you should be able to access a local Dgraph instance at `localhost:9080`. This is also the default configuration for Dgraph's adapter - you don't need to provide `GRAPH_SERVICE_HOST` explicitly.
```
Comment (CermakM) [inline README.rst]: Shouldn't it be `localhost:8080`? instead of `localhost:9080`?

Comment (fridex) [reply README.rst]: :9080 is correct.

File: README.rst (hunk 1 above)
Comment (fridex) [reply README.rst]: Nice, thanks! Fixed.

**Pull request merged**